improver_tests/set_up_test_cubes.py

```diff
@@ -317,40 +404,10 @@ def set_up_percentile_cube(
         percentiles (list or numpy.ndarray):
             List of int / float percentile values whose length must match the
             first dimension on the input data cube
-        name (str):
-            Variable standard name
-        units (str):
-            Variable units
-        spatial_grid (str):
-            What type of x/y coordinate values to use.  Default is "latlon",
-            otherwise uses "projection_[x|y]_coordinate".
-        time (datetime.datetime):
-            Single cube validity time
-        time_bounds (Iterable[datetime.datetime]):
-            Lower and upper bound on time point, if required
-        frt (datetime.datetime):
-            Single cube forecast reference time
-        include_scalar_coords (list):
-            List of iris.coords.DimCoord or AuxCoord instances of length 1.
-        attributes (dict):
-            Optional cube attributes.
-        standard_grid_metadata (str):
-            Recognised mosg__model_configuration for which to set up Met
-            Office standard grid attributes.  Should be 'uk_det', 'uk_ens',
-            'gl_det' or 'gl_ens'.
+        **kwargs:
```

Comment:
This `**kwargs` is much nicer than duplicating all the docstring and args across these functions.